flows/actions/testdata/add_contact_groups.json

```diff
@@ -11,7 +11,7 @@
                 }
             ]
         },
-        "validation_error": "no such group with UUID '33382939-babf-4982-9395-8793feb4e7c6'"
+        "validation_error": "missing dependencies: group[uuid=33382939-babf-4982-9395-8793feb4e7c6,name=Climbers]"
```

Comment:
this should never end up being a user facing error.. assuming we track dependencies correctly